src/JJMasterData.Core/DataDictionary/Dictionary.cs

```diff
@@ -1,22 +1,25 @@
-﻿using System.Runtime.Serialization;
+﻿using System;
+using System.Runtime.Serialization;
 using JJMasterData.Commons.Dao.Entity;
+using JJMasterData.Commons.Extensions;
+using JJMasterData.Commons.Language;
 
-namespace JJMasterData.Core.DataDictionary.DictionaryDAL;
+namespace JJMasterData.Core.DataDictionary;
 
 [DataContract(Name = "elementInfo")]
-public class DicParser
+public class Dictionary
```

Comment:
I think `Dictionary` is bad because .NET already have a `Dictionary` class heavily used. What about DataDictionary?

Reply:
It's true, I hadn't thought of that.
How about DicParser to MetaData?